data/json/items/armor/combat_exoskeleton.json

```diff
@@ -0,0 +1,471 @@
+        "moves": 1000
+      },
+      {
+        "pocket_type": "CONTAINER",
+        "holster": true,
+        "volume_encumber_modifier": 0,
+        "ablative": true,
+        "description": "Attachment points for exoskeleton torso armor.",
+        "flag_restriction": [ "ABLATIVE_EXOSKELETON_TORSO" ],
+        "max_contains_volume": "20000 ml",
+        "max_contains_weight": "20000 g",
+        "moves": 1000
+      },
+      {
+        "pocket_type": "CONTAINER",
+        "holster": true,
+        "volume_encumber_modifier": 0,
+        "ablative": true,
+        "description": "Attachment points for exoskeleton arm armor.",
+        "flag_restriction": [ "ABLATIVE_EXOSKELETON_ARMS" ],
+        "max_contains_volume": "20000 ml",
+        "max_contains_weight": "20000 g",
+        "moves": 1000
+      },
+      {
+        "pocket_type": "CONTAINER",
+        "holster": true,
+        "volume_encumber_modifier": 0,
+        "ablative": true,
+        "description": "Attachment points for exoskeleton leg armor.",
+        "flag_restriction": [ "ABLATIVE_EXOSKELETON_LEGS" ],
+        "max_contains_volume": "20000 ml",
+        "max_contains_weight": "20000 g",
+        "moves": 1000
+      }
+    ]
+  },
+  {
+    "id": "combat_exoskeleton_heavy",
+    "copy-from": "combat_exoskeleton_salvaged",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "combat exoskeleton mk1" },
+    "description": "Colloquially known as a 'tank suit' in the media, these first-generation exoskeletons were tested in military service a few years back and determined to be too expensive for regular use.  The exoskeleton itself is strapped on an environmental suit with an integrated life-support system.  A variety of of webbing, magnets, and clips allows the attachment of special armor plating onto the frame.",
+    "looks_like": "power_armor_heavy",
+    "price": 50000000,
+    "price_postapoc": 100000,
+    "extend": { "flags": [ "COMBAT_TOGGLEABLE" ] },
+    "ammo": "battery",
+    "use_action": {
+      "type": "transform",
+      "msg": "You turn the exoskeleton on.\n>>Initiating boot sequence…\n//Loading DoubleOS v1.39…\n//Loading calibration profile…\n//Activating atmospheric filtration…\n//Activating liquid cooling…\n//Activating sound dampeners…\n>>Boot successful.",
+      "target": "combat_exoskeleton_heavy_on",
+      "active": true,
+      "need_charges": 1,
+      "need_charges_msg": "The exoskeleton batteries are empty."
+    },
+    "relic_data": { "passive_effects": [ { "id": "combat_exoskeleton_heavy_active" } ] }
+  },
+  {
+    "copy-from": "combat_exoskeleton_heavy",
+    "id": "combat_exoskeleton_heavy_on",
+    "type": "TOOL_ARMOR",
+    "name": { "str": "combat exoskeleton mk1 (on)", "str_pl": "combat exoskeletons mk1 (on)" },
+    "extend": { "flags": [ "GAS_PROOF", "WATCH", "PARTIAL_DEAF", "TWO_WAY_RADIO" ] },
+    "power_draw": "200 W",
+    "revert_to": "combat_exoskeleton_heavy",
+    "use_action": {
+      "type": "transform",
```

Comment:
Actually, after a bit of testing it turns out I have 2 issues with the manage_exosuit menu:

1. The "execute which action" menu lists "manage exosuit modules" as an option, not ideal given this doesn't manage exosuit modules or an exosuit in the first place.
2. The manage exosuit modules menu lists the pockets(?) by the name of their flag, meaning I get a list that looks like EXO_HEAD_PLATE > EXO_TORSO_PLATE > EXO_ARM_PLATE etc. I'm not sure what to do there, change the name of the flags or how the menu itself lists what it's managing.

Reply:
Also, it breaks the tests due to npc not being able to combat_toggle items through the manage_exosuit menu.

Reply:
The UI isn't mandatory, but is helpful for incorporating anything beyond armor plate like power supplies or jetpacks or whatever. It can also be revisited late and have a similar version built for the core suits instead if it becomes necessary.

Reply:
Alright, I'll stash this for a later PR then, I'd rather be done with this one before messing with more code.